kedro/extras/extensions/ipython.py

```diff
@@ -49,6 +51,7 @@ def reload_kedro(path, env: str = None, extra_params: Dict[str, Any] = None):
 
     _remove_cached_modules(metadata.package_name)
 
+    configure_project(metadata.package_name)
```

Comment:
Removed `configure_project` from the session creation, but it's still needed by the ipython extension. So maybe this isn't that much of an improvement..

Reply:
I think that's fine, it makes it more explicit that `configure_project` is needed in all 3 main workflows atm: CLI, ipython, package mode.